inspire_query_parser/config.py

```diff
@@ -159,8 +159,8 @@
     'vol': 'volume',
 
     # Journal year
-    'journal-year': 'journal-year',
-    'jy': 'journal-year',
+    'journal-year': ('journal-year', 'publication_info.year'),
+    'jy': ('journal-year', 'publication_info.year'),
```

Comment:
why do you need `journal-year` here in the value? wouldn't it work with just
```suggestion
    'journal-year': 'publication_info.year',
    'jy': 'publication_info.year',
```